forc-plugins/forc-crypto/src/keccak256.rs

```diff
@@ -0,0 +1,8 @@
+use sha3::{Digest, Keccak256};
+
+/// Hashes a given data using Keccak256
+pub fn hash<T: Into<Vec<u8>>>(data: T) -> anyhow::Result<Vec<u8>> {
+    let mut hasher = Keccak256::new();
+    hasher.update(data.into());
+    Ok(hasher.finalize().to_vec())
```

Comment:
Passing `-1` does not work because the crate  will attempt to parse. I just fixed `""` parsing in cac394f58 (it was attempting to read from stdin when the first argument was empty. Now it'll do if the first argument is `-` or not given).

Reply:
However `echo -1 |  forc-crypto keccak256` works as expected.

Reply:
What about `forc crypto keccak256 "-1"`? Should it work for any string, in this case a string containing "-1"?